packages/trip-form/src/util.js

```diff
@@ -49,7 +47,56 @@ export function getModeString(modeObj) {
   return modeObj.mode || modeObj;
 }
 
-export function getTransitSubmodeOptions(icons, modes, selectedModes) {
+/**
+ * Of the specified companies, returns those that operate the specified modes.
+ * @param {*} companies The supported companies per OTP configuration.
+ * @param {*} modes The desired modes for which to get the operating companies.
+ * @returns An array of companies that operate the specified modes (should not be undefined as companies is an array).
+ */
+function getCompanies(companies, modes) {
+  return companies
+    .filter(
+      comp => comp.modes.split(",").filter(m => modes.includes(m)).length > 0
+    )
+    .filter(comp => hasRental(comp.modes) || hasHail(comp.modes));
+}
```

Comment:
I think this code `getCompanies` was just moved up in the file, which generally I would tend to avoid in a large PR like this... it just makes it seem like new code when it's not actually.

Reply:
That was enforced by es-lint I'm afraid.